### ISSUE TITLE
minor remaining issues caught by the cran winbuilder check

### ISSUE THREAD

**Issue by e-kotov** [rOpenSpain/spanishoddata#112]
1. ine.es and transportes.gob.es links give 403 forbidden when checked by CRAN even though the links actually work.
  - **solution:** will add a NOTE to the submission about that

2. Vignette inter-linking errors

```
Found the following (possibly) invalid file URIs:
  URI: flowmaps-static.html
    From: inst/doc/convert.html
  URI: flowmaps-interactive.html
    From: inst/doc/convert.html
  URI: converting-the-data-for-faster-analysis.html
    From: inst/doc/convert.html
  URI: flowmaps-static.html
    From: inst/doc/quick-get.html
  URI: flowmaps-interactive.html
    From: inst/doc/quick-get.html
  URI: convert.qmd
    From: README.md
```

To reduce package size, I have previously excluded some vignettes with lots of graphics from the package bundle. These are only available on the package website. This now causes warngings.

  - **solution:** add direct web links to where we expect the vignettes to be.

3. invalid DOI

```
Found the following (possibly) invalid DOIs:
  DOI: 10.32614/CRAN.package.spanishoddata
    From: inst/CITATION
    Status: 404
    Message: Not Found
```

  - **solution:** I will simply add a note that this is a prospective DOI that will be assigned anyway. I already did this with [`{flowmapblue}`](https://github.com/FlowmapBlue/flowmapblue.R) and CRAN reviewers were ok with that.

4. Typo for "Minstry" already fixed in the main branch.

Full log: https://win-builder.r-project.org/Hj7KAeNO6AE6/ + https://win-builder.r-project.org/Hj7KAeNO6AE6/00check.log

**Comment by Robinlovelace** [comment]
Good summary of the issues, great job on pushing for CRAN submission.

**Comment by e-kotov** [comment]
closed by #113
